src/store/Reducers/SystemPackageListStore.js

```diff
@@ -2,7 +2,8 @@ import { STATUS_LOADING, STATUS_REJECTED, STATUS_RESOLVED, storeListDefaults } f
 import { changeListParams, getNewSelectedItems } from '../../Utilities/Helpers';
 import * as ActionTypes from '../ActionTypes';
 
-export const SystemPackageListStore = (state = storeListDefaults, action) => {
+let stateInit = { queryParams: { filter: { updatable: 'true' } } };
```

Comment:
Just out of my curiosity, do we need string type of 'true' or bool type true

Reply:
as it is a value that is provided to "Radio" component by PF that accepts strings only, it is really a string that represents boolean